example/programs/topic-subscriber/src/models.rs

```diff
@@ -0,0 +1,21 @@
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
```

Comment:
Why does this struct derive Serialize? It’s unused, and the documentation md file says that only deserialization is in scope for the example.